.github/workflows/build.yaml

```diff
@@ -360,21 +356,3 @@ jobs:
             --title "Radius v${{ env.REL_VERSION }}" \
             --notes-file docs/release-notes/v${{ env.REL_VERSION }}.md \
             --verify-tag
-
-  delete_artifacts:
```

Comment:
For `we do not have storage limit anymore` - I might have missed this, could you share more details on this?

Reply:
This was because of private repo limitation. For public repo, storage is unlimited space. So we don't need to delete it anymore.